shared/types/submission.ts

```diff
@@ -126,23 +172,49 @@ export type StorageModeSubmissionStreamDto = z.infer<
   typeof StorageModeSubmissionStreamDto
 >
 
+export const MultirespondentSubmissionStreamDto =
+  MultirespondentSubmissionBase.pick({
+    submissionType: true,
+    form_fields: true,
+    form_logics: true,
+    encryptedSubmissionSecretKey: true,
+    encryptedContent: true,
+    verifiedContent: true,
+    version: true,
+  }).extend({
+    attachmentMetadata: z.record(z.string()),
```

Comment:
why not follow `attachmentMetadata: z.map(z.string(), z.string()).optional(),` in `MultirespondentSubmissionBase`?